docs/source/api/meshes/index.rst

```diff
@@ -7,3 +7,4 @@ Meshes
   zero_dimensional_submeshes
   one_dimensional_submeshes
   two_dimensional_submeshes
+  scikit_fem_submeshes_3d
```

Comment:
minor, but can we call this "three_dimensional_submeshes" for consistency with the other names